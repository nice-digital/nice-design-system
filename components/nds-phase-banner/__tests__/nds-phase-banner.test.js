"use strict";

describe("@nice-digital/nds-phase-banner", () => {
	test.todo("add phase banner component tests");
});
