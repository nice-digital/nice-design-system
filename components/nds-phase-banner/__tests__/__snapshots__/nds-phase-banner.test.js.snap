// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Phase Banner should match the default snapshot 1`] = `
<p
  className="phase-banner"
>
  <span
    className="phase-banner__tag"
  >
    <Tag
      alpha={true}
      impact={true}
    >
      Alpha
    </Tag>
  </span>
  <span
    className="phase-banner__label"
  >
    Some content
  </span>
</p>
`;
